ocs/defaults.py

```diff
@@ -7,19 +7,47 @@
 
 from ocs import constants
 
-STORAGE_API_VERSION = 'storage.k8s.io/v1'
-ROOK_API_VERSION = 'ceph.rook.io/v1'
-OCP_API_VERSION = 'project.openshift.io/v1'
-OPENSHIFT_REST_CLIENT_API_VERSION = 'v1'
-
-# Be aware that variables defined above and below are not used anywhere in the
-# config files and their sections when we rendering config!
-
-INSTALLER_VERSION = '4.1.2'
-CLIENT_VERSION = INSTALLER_VERSION
-ROOK_CLUSTER_NAMESPACE = 'openshift-storage'
-KUBECONFIG_LOCATION = 'auth/kubeconfig'  # relative from cluster_dir
-API_VERSION = "v1"
-CEPHFILESYSTEM_NAME = 'ocsci-cephfs'
+
+#############################################################################
+# Default Configuration - access those vars from config and not from here!  #
+#############################################################################
+ENV_DATA = {
```

Comment:
Instead of constants, As per some discussion last week, I think few said they belong to *defaults* and many of the current entries in *defaults* needs a new file name.

Reply:
@vasukulkarni not sure I follow here. I can move this config related default to diff module, or we can decide to have it still in yaml file, see my comment below. And discuss probably tomorrow.